code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -485,10 +485,10 @@ GLOBAL_LIST_INIT(metalhydrogen_recipes, list(
 	))
 
 /obj/item/stack/sheet/mineral/metal_hydrogen
-	name = "Metal Hydrogen"
+	name = "metal hydrogen"
```

Comment:
speaking of metal hydrogen, could you fix up the grammar on metallic hydrogen golems?